src/backend/marsha/core/utils/api_utils.py

```diff
@@ -1,7 +1,10 @@
 """Utils used by the api module."""
+from dataclasses import dataclass
+from functools import cache
```

Comment:
This import needs at least python 3.9
Can you try to use `mcr.microsoft.com/playwright:jammy` [here](https://github.com/openfun/marsha/blob/ab5e839fef64be491d9ebea69ebffecffc543bef/.circleci/config.yml) and [here](https://github.com/openfun/marsha/blob/ab5e839fef64be491d9ebea69ebffecffc543bef/.circleci/config.yml) please ?

Reply:
Marsha does not require Python >= 3.9?

Reply:
Indeed, marsha runs on 3.10, but our e2e testing images are still on 3.8.
We forgot to upgrade them I suppose…

Reply:
It does not work with `jammy`but the error is 

```
ModuleNotFoundError: No module named 'tzdata'
```

Reply:
For the time being, I can use 

``` python
try:
    from functools import cache
except ImportError:
    # For python < 3.9 we use the LRU cache
    from functools import lru_cache as cache
```

Or directly `lru_cache` what do you think ?

Reply:
I don't think we should adapt our codebase for an old testing image.
Please keep on using modern python code, as I'll look to the playwright problem.